snark-verifier-sdk/Cargo.toml

```diff
@@ -20,11 +20,11 @@ ark-std = { version = "0.3.0", features = ["print-trace"], optional = true }
 snark-verifier = { path = "../snark-verifier", default-features = false }
 
 # system_halo2
-halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2", tag = "v2023_04_20" } # not optional for now
+halo2_proofs = { git = "https://github.com/taikoxyz/halo2", tag = "v0.0.1" } # not optional for now
 halo2curves = { git = 'https://github.com/privacy-scaling-explorations/halo2curves', tag = "0.3.2" }  # must be same version as in halo2_proofs
 
 # loader_halo2
-halo2_wrong_ecc = { git = "https://github.com/privacy-scaling-explorations/halo2wrong", tag = "v2023_04_20", package = "ecc", optional = true }
+halo2_wrong_ecc = { git = "https://github.com/taikoxyz/halo2wrong", branch = "a6-integration", package = "ecc", optional = true }
```

Comment:
I don't see a PR open to merge this in main. Does this need to be a different branch?

Reply:
I mean, do we want to keep the main same as PSE? If no, this can goes to the main :)

Reply:
This PR is merging changes into main on this repo, so seems like we should do it on main everywhere (or nowhere)?